packages/nc-gui-v2/components/smartsheet/Cell.vue

```diff
@@ -17,14 +18,25 @@ interface Emits {
 
 const { column, ...props } = defineProps<Props>()
 
-const emit = defineEmits(['update:modelValue', 'save'])
+const emit = defineEmits(['update:modelValue', 'save', 'navigate', 'cancel'])
 
 provide(ColumnInj, column)
 
-provide(EditModeInj, toRef(props, 'editEnabled'))
+provide(
+  EditModeInj,
+  computed({
+    get() {
+      return props?.editEnabled
+    },
+    set() {
+      return emit('cancel')
+    },
+  }),
+)
```

Comment:
Why not use another `vModel` instead of a regular computed prop? You could then just bind it as `<Cell v-model:edit-enabled="prop" />`

Reply:
Ya should be fine, In parent I can do something like

```
js:edit-enabled="editEnabled && selected.col === colIndex && selected.row === rowIndex"
@update:edit-enabled="editEnabled  = false"
```

Reply:
If you use `v-model:edit-enabled` the `@update:edit-enabled` binding should be unnecessary
`v-model` automatically binds the `update` event

Reply:
But there is an extra `&& selected.col === colIndex && selected.row === rowIndex` part